### PR TITLE
Add `Byte Converter` to converter extension

### PR DESCRIPTION
**Pull request by Saafo**
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by Saafo** [comment]
@koinzhang Sorry for the interruption, I've add a `byte converter` to the `Converter` extension managed by you, I'm not sure whether it's acceptable. Please have a look.

image

Besides, there's something that are not sure:

1. I'm not sure whether to add a `byte converter` pic to metadata, so we can see the snapshot on the store.
2. Currently I didn't add any `ActionPanel`, I'm not sure whether it's required, personally I won't use the action panel.
3. There's no icon for the `byte converter` now, I think it's better to keep a similar icon style with the existing two converters. I'm not sure how you generated them.

**Comment by koinzhang** [comment]
Hi @Saafo. Thank you very much for your contribution, I'm sure adding this command will be very useful.
I hope you can complete the ActionPanel (and update the raycast/api version in package.json to the latest by the way).
I will provide you with a command icon.
[byte-converter-icon.png.zip](https://github.com/raycast/extensions/files/11411618/byte-converter-icon.png.zip)

**Comment by koinzhang** [comment]
Hi @Saafo. 
1. If you want to update the metadata and wish you could use a uniform background, you can recreate the three metadata, Raycast comes with a screenshot function in Advanced->WIndow Capture. It would be nice if you could use Raycast Wallpaper as the background.
2. If you are sure to add Byte Converter command, please update the extension introduction, README, CHANGELOG and other information at the same time
[Glass Rainbow.png.zip](https://github.com/raycast/extensions/files/11411631/Glass.Rainbow.png.zip)

**Comment by Saafo** [comment]
Ok, I'll do these next week!

**Comment by Saafo** [comment]
Hi @koinzhang,
1. I've complete the `ActionPanel` part, including copy the best-expressive unit, current unit and all units.
2. I've updated the raycast/api version to `1.51.0`.
3. I've replaced the command icon with the one you provided.
4. I've created a metadata picture follow your sweet instruction(thanks for the hint!) as `metadata/conveter-3.png`, please have a check.
5. I've update the descriptions for Byte converter in package.json, README and CHANGELOG, please have a check.

**Comment by koinzhang** [comment]
Hi @pernielsentikaer LGTM.
Hi @Saafo Thanks again for your contribution and I look forward to more contributions from you in the Raycast community!

**Comment by Saafo** [comment]
hello @pernielsentikaer, can we merge this pr now?

File: extensions/converter/src/byte-converter.tsx [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+          <Action.CopyToClipboard title={`Copy ${getFocusedValueExpression()}`} content={getFocusedValueExpression()} />
+          <ActionPanel.Section>
+            {units.map((_, index) => {
+              const keyEquivalent = KeyEquivalentByNumber(index + 1);
+              return (
+                <Action.CopyToClipboard
+                  key={index} // to make key unique to avoid warning
+                  title={`Copy ${getExpressionAtIndex(index)}`}
+                  content={getExpressionAtIndex(index)}
+                  shortcut={keyEquivalent ? { modifiers: ["cmd"], key: keyEquivalent } : undefined}
```
Comment (pernielsentikaer) [inline extensions/converter/src/byte-converter.tsx:110]: cmd+1/2/3/4/5/6/7/8/9/0 are already used by Raycast whenever a list is present, so this shortcut should be changed to something else (e.g. cmd+shift+1)

**Review by pernielsentikaer — APPROVED** [review]

**Pull request merged**